bm25s/stopwords.py

```diff
@@ -33,3 +33,2774 @@
     "will",
     "with",
 )
+
```

Comment:
It would be great to add a notice that the stopwords were taken from NLTK, and link to the exact file/page where they were retrieved!

```suggestion
# The stopwords below are retrieved from NLTK: ...
```

Reply:
done